internal/provider/resource_incident_impact_source.go

```diff
@@ -317,6 +322,26 @@ Options: ALL, P1, P2, P3, P4, P5. Defaults to ALL`,
 					},
 				},
 			},
+			"clubhouse_input": {
+				Description: "Clubhouse input",
+				Type:        schema.TypeList,
+				MaxItems:    1,
+				Optional:    true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"remote_query": {
+							Type:        schema.TypeString,
+							Required:    true,
```

Comment:
I made it required in related [PR](https://github.com/sleuth-io/sleuth/pull/9325/files#diff-efc1675187620595b0844197a25c35eac9b6df752f9182e5cffddee6f27a8489R3478) because it's required on the [model](https://github.com/sleuth-io/sleuth/blob/master/sleuth/apps/clubhouse/impact.py#L65).